Library/Homebrew/download_strategy.rb

```diff
@@ -330,7 +330,7 @@ def _fetch
 class NoUnzipCurlDownloadStrategy < CurlDownloadStrategy
   def stage
     UnpackStrategy::Uncompressed.new(cached_location)
-                                .extract(basename: basename_without_params)
+                                .extract(basename: basename_without_params, verbose: ARGV.verbose? && !shutup)
```

Comment:
This line is a bit long. `verbose` could be extracted to be a variable, perhaps?